src/server/views/confirmation.html

```diff
@@ -10,6 +10,7 @@
     {{ govukPanel({
       titleText: "Application complete"
     }) }}
+    {% include "help/what-happens-next/" + slug + ".html" ignore missing %}
```

Comment:
Can we indent this and drop in `pageTitle` from above?

```suggestion
      {{ govukPanel({
        titleText: pageTitle
      }) }}
    
      {% include "help/what-happens-next/" + slug + ".html" ignore missing %}
```

Reply:
Done, cheers